security/accesscontrol.rst

```diff
@@ -330,115 +124,6 @@ The other option would be to follow `Quickstart: Deploy Azure API for FHIR using
-
-      :Request: ``GET [base]/Patient?link:Patient.identifier=456``
-      :Type-Access: User must have read access to Patient, otherwise a 401 is returned.
-      :Compartment: Is applied to both Patient and link. In pseudo code: ``GET [base]/Patient?link:(Patient.identifier=456&Patient.identifier=123)&identifier=123`` In this case there will probably be no results.
-
-#. Read: Is evaluated as a Search, but implicitly you only specify the _type and _id search parameters.
-#. VRead: If a user can Read the current version of the resource, he is allowed to get the requested version as well.
-#. Create
-
-   a. Create on the compartment type
-
-      :Request: ``POST [base]/Patient``
-      :Type-Access: User must have write access to Patient. Otherwise a 401 is returned.
-      :Compartment: A Search is performed as if the new Patient were in the database, like in case 1.a. If it matches the compartment filter, the create is allowed. Otherwise a 401 is returned.
-
-   #. Create on a type related to compartment
-
-      :Request: ``POST [base]/Observation``
-      :Type-Access: User must have write access to Observation. Otherwise a 401 is returned. User must also have read access to Patient, in order to evaluate the Compartment.
-      :Compartment: A Search is performed as if the new Observation were in the database, like in case 1.b. If it matches the compartment filter, the create is allowed. Otherwise a 401 is returned.
-
-   #. Create on a type not related to compartment
-
-      :Request: ``POST [base]/Organization``
-      :Type-Access: User must have write access to Organization. Otherwise a 401 is returned.
-      :Compartment: Is not evaluated.
-
-#. Update
-
-   a. Update on the compartment type
-
-      :Request: ``PUT [base]/Patient/123``
-      :Type-Access: User must have write access *and* read access to Patient, otherwise a 401 is returned.
-      :Compartment: User should be allowed to Read Patient/123 and Create the Patient provided in the body. Then Update is allowed.
-
-   #. Update on a type related to compartment
-
-      :Request: ``PUT [base]/Observation/xyz``
-      :Type-Access: User must have write access to Observation, and read access to both Observation and Patient (the latter to evaluate the compartment)
-      :Compartment: User should be allowed to Read Observation/123 and Create the Observation provided in the body. Then Update is allowed.
-
-#. Delete: Allowed if the user can Read the current version of the resource, and has write access to the type of resource.
-#. History: Allowed on the resources that the user is allowed to Read the current versions of (although it is theoretically possible that an older version would not match the compartment). 
-
-.. note:: A conditional create, update or delete (see the `FHIR http specification <https://hl7.org/fhir/http.html>`_), requires read permissions on the condition. Therefore, ``user/*.write`` will usually require additional ``read`` scopes.
-
-Testing
```

Comment:
This section is removed and I cannot find it anywhere else, is it intentional?

Reply:
Yes, the Firely Auth section has a testing section already.